APMrover2/Parameters.h

```diff
@@ -317,8 +317,8 @@ class Parameters {
 
         // PID controller    initial P        initial I        initial D        initial imax
         //-----------------------------------------------------------------------------------
-        pidSpeedThrottle    (0.7,             0.2,             0.2,             4000)
-        {}
+        pidSpeedThrottle(0.7,             0.2,             0.2,             4000)
```

Comment:
variable names comment, value columns match.
looked i understand.
this change is unmatched columns.
if value columns is good, variable names comment is columns matched.